papers/ring_vrf/overview.tex

```diff
@@ -34,8 +34,8 @@ \section{Protocol overview}
 
 % As an instructive but insecure over simplification, 
 At their simplest,
-ring VRFs' other algorithms might operate directly upon
- the plausible signer set \ring, like:
+ring VRFs' other algorithms operate directly upon
+ the alleged signer set \ring, like:
```

Comment:
alleged has a different meaning that does not quite fit.

Reply:
No problem. Reverted to plausible.